emails/models.py

```diff
@@ -91,6 +91,9 @@ class Profile(models.Model):
     last_account_flagged = models.DateTimeField(blank=True, null=True, db_index=True)
     num_email_forwarded_in_deleted_address = models.PositiveIntegerField(default=0)
     num_email_blocked_in_deleted_address = models.PositiveIntegerField(default=0)
+    num_level_one_trackers_blocked_in_deleted_address = models.PositiveIntegerField(
```

Comment:
This worked, tested as:

1. Apply the migration
2. Check out `main`. I should check out `v3.5.21`, but `main` should work too
3. Load the site, create a relay alias

Previously, it fails, because of the `NOT NULL constraint failed`. Now it succeeds, and sets `num_level_one_trackers_blocked` to `NULL`. 

However, the profile breaks when switching back to this branch, because it is attempting to add 0 and `None` and failing. The code needs to interpret a `None` as 0.

Maybe we'll try the database default next time...